src/server/pkg/metrics/segment.go

```diff
@@ -23,6 +24,28 @@ func reportSegment(metrics *Metrics) {
 		},
 	})
 	if err != nil {
-		lion.Errorf("error reporting to Segment: %s", err.Error())
+		lion.Errorf("error reporting cluster metrics to Segment: %s", err.Error())
+	}
+}
+
+func identifyNewUser(userID string) {
```

Comment:
I simplified the name to just `identifyUser()` ... which can be called as many times as we like, but does need to get called before any `Track()` calls